src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,190 @@
+
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+
+    private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+    public static void deleteFile(String filePath) throws IOException {
+        Files.delete(Paths.get(filePath));
+    }
+
+    public static void sayHello() {
+        System.out.println(HORIZONTAL + "Hello! I'm Duke\n" + "What can I do for you?\n" + HORIZONTAL);
+    }
+
+    public static void sayBye() {
+        System.out.println(HORIZONTAL + "Bye. Hope to see you again soon!\n" + HORIZONTAL);
+    }
+
+    public static void addToDo(String line) throws DukeException {
+        String task = line.trim().substring(INDEX_OF_TO_DO_TASK);
+        if (task.equals("")) {
+            throw new DukeException();
+        }
+        tasks.add(new ToDo(task));
+        System.out.println(HORIZONTAL + "Got it. I've added this task: \n  " + tasks.get(tasks.size() - 1) +
+                "\n" + "Now you have " + tasks.size() + " task" + (tasks.size() == 1? " " : "s ") +
+                "in the list. \n" + HORIZONTAL);
+    }
+
+    public static void addDeadline(String line) throws DukeException {
+        if (!line.contains("/by")) {
+            throw new DukeException();
+        }
+        int slash = line.indexOf("/by");
+        String task = line.trim().substring(INDEX_OF_DEADLINE_TASK, slash);
+        if (task.equals("")) {
+            throw new DukeException();
+        }
+        String dueDate = line.trim().substring(slash + 3);
+        if (dueDate.equals("")) {
+            throw new DukeException();
+        }
+        tasks.add(new Deadline(task, dueDate));
+        System.out.println(HORIZONTAL + "Got it. I've added this task: \n  " + tasks.get(tasks.size() - 1) +
+                "\n" + "Now you have " + tasks.size() + " task" + (tasks.size() == 1? " " : "s ") +
+                "in the list. \n" + HORIZONTAL);
+    }
+
+    public static void addEvent(String line) throws DukeException {
+        if (!line.contains("/at")) {
+            throw new DukeException();
+        }
+        int slash = line.indexOf("/at");
+        String task = line.trim().substring(INDEX_OF_EVENT_TASK, slash);
+        if (task.equals("")) {
+            throw new DukeException();
+        }
+        String eventTime = line.trim().substring(slash + 3);
+        if (eventTime.equals("")) {
+            throw new DukeException();
+        }
+        tasks.add(new Event(task, eventTime));
+        System.out.println(HORIZONTAL + "Got it. I've added this task: \n  " + tasks.get(tasks.size() - 1) +
+                "\n" + "Now you have " + tasks.size() + " task" + (tasks.size() == 1? " " : "s ") +
+                "in the list. \n" + HORIZONTAL);
+    }
+
+    public static void addTask(String line) {
+        if (line.contains(TO_DO)) {
```

Comment:
Interesting way of parsing your query, what does your code work for "hello todo more CS2113 work"?
Same for the subsequent lines